sdk/messaging/eventgrid/azsystemevents/CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 0.4.4 (Unreleased)
+## 0.4.4 (2024-11-19)
```

Comment:
Shouldn't this be 0.5.0? There are breaking changes

Reply:
Good point, I'll update it.